ocaml/typing/typemod.ml

```diff
@@ -3307,6 +3307,22 @@ let gen_annot outputprefix sourcefile annots =
   Cmt2annot.gen_annot (Some (outputprefix ^ ".annot"))
     ~sourcefile:(Some sourcefile) ~use_summaries:false annots
 
+let cms_register_toplevel_struct_attributes ~sourcefile ~uid ast =
+  (* Cms files do not store the typetree. Toplevels attributes are
+    as they are not explicitely associated with a uid so we need
+    to manually register them. *)
+  let attributes =
+    List.filter_map
+      (function
+        | { pstr_desc = Pstr_attribute attr; _ } -> Some attr
+        | _ -> None
+      )
+      ast
+  in
```

Comment:
What is the expected result in merlin if there are multiple doc comments picked up here?

Reply:
Only the first one will be returned by merlin